src/core/observables/PersistenceAngles.hpp

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright (C) 2019 The ESPResSo project
+
+This file is part of ESPResSo.
+
+ESPResSo is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+ESPResSo is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifndef OBSERVABLES_PERSISTENCEANGLES_HPP
+#define OBSERVABLES_PERSISTENCEANGLES_HPP
+
+#include "PidObservable.hpp"
+#include <utils/Vector.hpp>
+
+#include <cmath>
+#include <vector>
+
+namespace Observables {
+
+/** Calculate bond angles between in a polymer.
+ *  For @f$ n @f$ bonded particles, return the @f$ n-2 @f$ angles along the
+ *  chain that are seperated by 1 bond, 2 bonds, ..., in radians.
```

Comment:
```suggestion
 *  chain that are separated by 1 bond, 2 bonds, ..., in radians.
```
also, at index `i=0`, `i=1`, ..., the angle is for particles separated by 2 bonds, 3 bonds, ... (or equivalently for bonds separated by 0 bond, 1 bond, ..., as you wrote in analysis.rst); the current formulation seems ambiguous